internal/receiver/smartagentreceiver/output.go

```diff
@@ -162,37 +162,37 @@ func getLoneSFxExporter(host component.Host, exporterType collectorConfig.DataTy
 
 }
 
-func (output *Output) AddDatapointExclusionFilter(filter dpfilters.DatapointFilter) {
+func (output *output) AddDatapointExclusionFilter(filter dpfilters.DatapointFilter) {
```

Comment:
```suggestion
func (out *output) AddDatapointExclusionFilter(filter dpfilters.DatapointFilter) {
```
just to avoid overriding the type